src/components/sidebar/FAB.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * @name FAB
+ * @summary Renders a FloatingActionButton
+ * @return {jsx} React FAB
+ */
+const Fab = () => (
+  <div className='fab'>
+    <div className='line' />
```

Comment:
Your class names do not follow BEM. Ideally, this should be something like.
```
<div className='fab'>
  <div className='fab__line fab__line--1' />
  <div className='fab__line fab__line--2' />
  <div className='fab__line fab__line--3' />
</div>
```
In the class name `fab__line--3`, Block = 'fab', element = 'line' and modifier = '3'.
The `fab__line--3` would then be used to set different widths for the lines, and 'fab__line' would be used to set rules that cut across the lines.